annlite/executor.py

```diff
@@ -17,6 +20,7 @@ class AnnLiteIndexer(Executor):
     def __init__(
         self,
         dim: int = 0,
+        data_path: str = './workspace',
```

Comment:
no, the `data_path` can be optional. If not provided by user, then we can use meta param `self.workspace`.